### PR TITLE
Adding nxos_evpn_global module

### PR DESCRIPTION
**Pull request by GGabriele**
##### ISSUE TYPE
- New Module Pull Request
##### COMPONENT NAME

nxos_evpn_global
##### ANSIBLE VERSION

```
ansible 2.2.0
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/usr/share/ansible/']
```
##### SUMMARY

A new module to handles the EVPN control plane for VXLAN.

### CODE REVIEW

**Comment by gregdek** [comment]
Thanks again to @GGabriele. This module is going into community review.

We notice that you are one of the maintainers (@privateip, @GGabriele, @gundalow) of this module, so when you think it's ready, please comment "shipit" and we will consider it for merging.

[This message brought to you by your friendly Ansibull-bot.]

File: network/nxos/nxos_evpn_global.py [hunk 1]
```diff
@@ -0,0 +1,319 @@
...
+version_added: "2.2"
+short_description: Handles the EVPN control plane for VXLAN.
+description:
+    - Handles the EVPN control plane for VXLAN.
+author: Gabriele Gerbino (@GGabriele)
+extends_documentation_fragment: nxos
+options:
+    nv_overlay_evpn:
+        description:
+            - EVPN control plane
```
Comment (gundalow) [inline network/nxos/nxos_evpn_global.py]: Missing fullstop

**Comment by gundalow** [comment]
I didn't notice this in the other modules, however I believe it applies to all the `nxos_*` modules

Any option you have defined in `extends_documentation_fragment: nxos` should be defined in `module_utils` and not in every module under `argument_spec`. This allow everything to be kept in sync as well as reducing the amount of code in the modules

**Comment by gundalow** [comment]
needs_revision

**Comment by gregdek** [comment]
Thanks @GGabriele for this PR. This PR requires revisions, either because it fails to build or by reviewer request. Please make the suggested revisions. When you are done, please comment with text 'ready_for_review' and we will put this PR back into review.

[This message brought to you by your friendly Ansibull-bot.]

**Pull request merged**